tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt

```diff
@@ -25,7 +25,6 @@
 /obj/item/pda : /obj/item/modular_computer/tablet/pda
 /obj/item/pda/medical : /obj/item/modular_computer/tablet/pda/medical
 /obj/item/pda/paramedic : /obj/item/modular_computer/tablet/pda/paramedic
-/obj/item/pda/virologist : /obj/item/modular_computer/tablet/pda/virologist
```

Comment:
Don't remove this, it's an important migration for legacy reasons

Reply:
Reverted

Reply:
It would need to point to a typepath that actually exists though